src/macros.jl

```diff
@@ -211,6 +222,7 @@ macro testitem(nm, exs...)
     retries = 0
     tags = Symbol[]
     setup = Any[]
+    _quote = false
```

Comment:
Maybe put the explanation comment here in addition to / instead of in the definition?